services/stats/stats_test.go

```diff
@@ -32,12 +32,6 @@ func Test_Measurement_Invalid_Operations(t *testing.T) {
 		require.Panics(t, func() {
 			s.NewStat("test", stats.CountType).Observe(1.2)
 		})
-		require.Panics(t, func() {
-			s.NewStat("test", stats.CountType).Start()
-		})
-		require.Panics(t, func() {
-			s.NewStat("test", stats.CountType).End()
-		})
```

Comment:
shouldn't these be replaced with `RecordDuration`? Same comment applies for all other invalid operations tests where `Start` & `End` have been removed.